### PR TITLE
 - Adding basic instructions to harden a dockerized production instance

### PR DESCRIPTION
**Pull request by afabiani**
This PR adds some basic steps to manage a geonode-project based production instance by ensuting it is protected and stable.

### CODE REVIEW

File: install/basic/index.rst [hunk 1]
```diff
@@ -512,3 +514,351 @@ Whenever you change someting on :guilabel:`.env` file, you will need to rebuild
...
+
+This means that your JVM will be started with Xms amount of memory and will be able to use a maximum of Xmx amount of memory. Above will start a JVM like with 2 GB of memory and will allow the process to use up to 4 GB of memory. You need to adjust this value depening on your availabnle RAM.
+
+``-DGEOSERVER_CSRF_DISABLED (True)``
+
+The GeoServer web admin employs a CSRF (Cross-Site Request Forgery) protection filter that will block any form submissions that didn’t appear to originate from GeoServer. This can sometimes cause problems for certain proxy configurations. You can disable the CSRF filter by setting the GEOSERVER_CSRF_DISABLED property to true.
+https://docs.geoserver.org/stable/en/user/security/webadmin/csrf.html
+
+
+Whenever you need to change one of more of the JVM options, you will need to restart the GeoServer Docker container.
```
Comment (t-book) [inline install/basic/index.rst:554]: Whenever you need to change some of the other available JVM options, you will need to restart the GeoServer Docker container.

?

**Comment by t-book** [comment]
Super @afabiani that's a great contribution. Only one little suggestion regarding `one of more of the JVM options`

**Comment by t-book** [comment]
by the way. I think it would be nice to set geofence rules storage as standard to PostGIS or configurable by docker-compose. maybe something for a code sprint ...

File: install/basic/index.rst (hunk 1 above)
Comment (afabiani) [reply install/basic/index.rst:554]: Typo "one or more"

**Comment by afabiani** [comment]
@t-book that was my idea too... just need to find some time to do it. Code sprint would be a good occasion,

**Pull request merged**